### PR TITLE
Added typescript

### PR DESCRIPTION
**Pull request by shanm-16**

### CODE REVIEW

**Comment by aryanguptaSG** [comment]
@shanm-16 add dist in gitignore .

File: Backend/src/service/facebook.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
+const facebookConfig = require('../config/facebook')
+const axiosModule = require('axios')
+
+
+
+let userTokens = ""
+
```
Comment (shiva4866) [inline Backend/src/service/facebook.ts:7]: make class instead

File: Backend/src/service/facebook.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+function auth(request : {},response : any){
+    return response.redirect('https://www.facebook.com/v15.0/dialog/oauth?response_type=code&client_id=575427578042258&redirect_uri=https://57ef-2401-4900-1cb9-a1d9-8c5e-2e5c-b745-5c45.ngrok-free.app/auth/callback')
+}
+
+function authCallback(request : any,response : any){
+    const code = request.query.code;
+
+
+      axiosModule.post('https://graph.facebook.com/v15.0/oauth/access_token?code=' + code + '&client_id=' + facebookConfig.client_id + '&client_secret=' + facebookConfig.client_secret + '&redirect_uri=' + facebookConfig.redirect_uri)
```
Comment (shiva4866) [inline Backend/src/service/facebook.ts:16]: move harcoded variables to constants

File: Backend/src/server.ts [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+app.listen(port, () => {
+  console.log(`Server is running on http://localhost:${port}`);
+});
+
+
+app.listen(port, () => {
+    console.log(`Server is running on http://localhost:${port}`);
+  });
+
+  app.get('/post', async (req: any, res: any) => {
```
Comment (shiva4866) [inline Backend/src/server.ts:21]: don't do write operation in get API.
Also don't write APIs here

**Pull request merged**